src/box/box.cc

```diff
@@ -1077,6 +1077,20 @@ box_check_sql_cache_size(int size)
 	return 0;
 }
 
+static void
+box_check_allocator(void)
+{
+	const char *allocator = cfg_gets("memtx_allocator");
+	if (allocator == NULL ||
```

Comment:
Default value is "small". Could it really become NULL?

Reply:
Fixed